### PR TITLE
Improve description for yellow-colored runs

### PR DESCRIPTION
**Pull request by mribeirodantas**
The description for color runs is not clear enough for the yellow color.
I don't think my suggestion is great either, but it's a start for a discussion.
Besides, isn't it actually orange?!

### CODE REVIEW

**Comment by mribeirodantas** [comment]
This looks orange to me 😅 

https://tower.nf/orgs/nf-core/workspaces/AWSmegatests/watch/25oeKsVLIpSZ3I

File: docs/monitoring/overview.md [hunk 1]
```diff
@@ -16,8 +16,8 @@ The colors signify the completion status:
 
   - **Blue** are running.
   - **Green** are successfully executed.
-  - **Yellow** are successfully executed where some tasks failed.
   - **Red** are jobs where at least one task fully failed.
+  - **Yellow** are successfully executed with at least a failed task that had ignore as error strategy.
```
Comment (bentsherman) [inline docs/monitoring/overview.md:20]: You should then clarify that red status means the workflow had a "terminate" error strategy. Also I would leave the order as it was.

Comment (mribeirodantas) [reply docs/monitoring/overview.md:20]: Good points! Agreed.

File: docs/monitoring/overview.md [hunk 2]
```diff
@@ -16,8 +16,8 @@ The colors signify the completion status:
 
   - **Blue** are running.
   - **Green** are successfully executed.
-  - **Yellow** are successfully executed where some tasks failed.
-  - **Red** are jobs where at least one task fully failed.
+  - **Yellow** are successfully executed with at least a failed task that had ignore as error strategy.
+  - **Red** are jobs where at least one task fully failed and had terminate as error strategy.
```
Comment (bentsherman) [inline docs/monitoring/overview.md]:
```suggestion
  - **Yellow** are successfully executed where at least one task failed with an "ignore" error strategy.
  - **Red** are successfully executed where at least one task failed with a "terminate" error strategy.
```

**Review by bentsherman — APPROVED** [review]

**Pull request merged**